TicketsDemo/App_Data/log.txt

```diff
@@ -6,3 +6,6 @@ Info[10.07.2014 14:18:34]: ticket TicketsDemo.Data.Entities.Ticket created
 Info[10.07.2014 14:20:50]: reservation TicketsDemo.Data.Entities.Reservation created
```

Comment:
log file shall be added to .gitignore as application resources are NOT a part of actual application.

In order to delete it from remote repository, do these steps:

1) delete the file and commit changes
2) add file to gitignore and commit changes

Reply:
Do the same with `UpgradeLog.htm`  as it is an artifact, created by visual studio and is NOT a part of the project codebase